src/components/axes/timeAxis.ts

```diff
@@ -207,6 +202,19 @@ export module Axis {
         return this._possibleTimeAxisConfigurations;
       }
       this._possibleTimeAxisConfigurations = configurations;
+      this._numTiers = _Util.Methods.max(this._possibleTimeAxisConfigurations.map((config: TimeAxisConfiguration) => config.length), 0);
```

Comment:
Because it makes sense now. The number of tiers is a property of the axis, which we may want to query in the future (or test).